hack/release.sh

```diff
@@ -51,6 +51,10 @@ GCR_KEY_FILE="${GCR_KEY_FILE:-}"
 GOPROXY="${GOPROXY:-https://proxy.golang.org}"
 BUILD_RELEASE_TYPE="${BUILD_RELEASE_TYPE:-}"
 
+# CUSTOM_REPO_FOR_GOLANG can be used to pass custom repository for golang
+# builder image. Ensure it ends with a '/'.
+GOLANG_IMAGE=${CUSTOM_REPO_FOR_GOLANG}golang:1.16
```

Comment:
can you provide an example in the comment for `CUSTOM_REPO_FOR_GOLANG`

Reply:
done